internal/executor/instance/container.go

```diff
@@ -18,6 +18,13 @@ func (inst *ContainerInstance) Run(ctx context.Context, config *RunConfig) (err
 		return err
 	}
 	defer func() {
+		if config.DockerOptions.NoCleanup {
+			config.Logger.Debugf("not cleaning up working volume %s, don't forget to remove it with \"docker volume rm %s\"",
```

Comment:
Should probably be Info level so it's visible for a user
```suggestion
			config.Logger.Infof("not cleaning up working volume %s, don't forget to remove it with \"docker volume rm %s\"",
```